src/components/landing/MobileNavigation.tsx

```diff
@@ -38,7 +51,7 @@ const MobileNavigation = ({ mainNavLinks, onMobileMenuToggle }: MobileNavigation
                   key={link.label}
                   href={link.href}
                   onClick={onMobileMenuToggle}
-                  className="text-dark-text-primary hover:bg-dark-bg-tertiary -mx-3 block rounded-lg px-3 py-2 text-base leading-7 font-semibold"
+                  className={`-mx-3 block rounded-lg px-3 py-2 text-base leading-7 font-semibold ${textPrimary} ${hoverBg}`}
```

Comment:
here seems to be a typo?
```suggestion
                  className={`mx-3 block rounded-lg px-3 py-2 text-base leading-7 font-semibold ${textPrimary} ${hoverBg}`}
```

Please check